### DISCUSSION TITLE
Question on Home Pro support

### DISCUSSION THREAD

**Discussion by pcb1962** [BottlecapDave/HomeAssistant-OctopusEnergy#924] [General]
Hi Dave, if I configure my Home Pro in the latest release will data from the Home Pro replace the data that is currently sourced from the Home Mini or will it create new entities alongside? 
It would be nice to be able to compare the quality of data from the two before committing to the Home Pro, which with the current mqtt  source (from neilkplugins) is seeing frequent dropouts.

**Comment by BottlecapDave** [comment]
Hello. As per the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/account/#settings), it replaces the source of existing entities.

I am also seeing dropouts in my viewing (which is why it's experimental), which seems to be the pro disconnecting from the meter, rather than HA not connecting to the pro.
